tools/AnalyzeDeps/AnalyzeDeps.ps1

```diff
@@ -0,0 +1,173 @@
+<#
```

Comment:
We are moving most of these type of things to the eng directory, in fact we plan to delete the tools directory soon so we we will need to move this.